src/app/containers/Toggle/toggleUtils.test.js

```diff
@@ -0,0 +1,28 @@
+import isToggleEnabled from './toggleUtils';
```

Comment:
Might there be some confusion that this is not imported like

```js
import { isToggleEnabled } from './toggleUtils';
```

Reply:
could import it that that way, however eslint then complains about preferring a default export. I am happy either way.

Reply:
Yeah I think either override the eslint rule and export as `{ isToggleEnabled }` or make a `utils` folder and add a `isToggleEnabled.js` (which i prefer)